docker-compose.ci.yml

```diff
@@ -1,4 +1,17 @@
 version: '2.3'
 services:
   django:
-    image: "open-apparel-registry-api:${GIT_COMMIT:-latest}"
+    image: "openapparelregistry:${GIT_COMMIT:-latest}"
+
+  terraform:
+    image: quay.io/azavea/terraform:0.11.11
+    volumes:
+      - ./:/usr/local/src
+      - $HOME/.aws:/root/.aws:ro
+    environment:
+      - AWS_PROFILE=${AWS_PROFILE:-open-apparel-registry}
+      - GIT_COMMIT=${GIT_COMMIT:-latest}
+      - OAR_DEBUG=1
+      - OAR_SETTINGS_BUCKET=${OAR_SETTINGS_BUCKET:-openapparelregistry-staging-config-us-east-1}
```

Comment:
It might be valuable to skip the inheritance of `OAR_SETTINGS_BUCKET` here and just default to the `openapparelregistry-staging-config-us-east-1` literal. `OAR_SETTINGS_BUCKET` is set to `openapparelregistry-development-config-us-east-1` in the virtual machine.

Reply:
Agreed.